File: Front/ProEventos-App/src/app/models/Speaker.ts
import { SocialNetwork } from "./SocialNetwork";

export interface Speaker {
    Id: number;
    Name: string;
    MiniResume: string;
    ImageURL: string;
    Phone: string;
    Email: string;
    SocialNetworks: SocialNetwork[];
    SpeakersEvents: Speaker[];
}

File: Back/src/ProEventos.Domain/Speaker.cs
using System.Collections.Generic;

namespace ProEventos.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniResume { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<SpeakerEvent> SpeakersEvents { get; set; }

    }
}